### PR TITLE
feat(*): make multiple plugins support the RPC builder

### PR DESCRIPTION
**Pull request by iawia002**
**What this PR does / why we need it**:

Other plugins like metrics rely on configuring the middleware for the root path (/) to automatically configure the middleware for all paths which doesn't work for the RPC API style, which requires manual configuration of middleware and descriptor for the RPC style server.

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #382 

**Special notes for your reviewer**:

/cc @xpofei 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: utils/api/definitions.go [hunk 1]
```diff
@@ -100,7 +99,7 @@ type Definition struct {
 // NewDefinition creates openapi.Definition from definition.Definition.
 func NewDefinition(tc *TypeContainer, d *definition.Definition, apiStyle string) (*Definition, error) {
```
Comment (xpofei) [inline utils/api/definitions.go:100]: 为啥不直接传入 APIStyle 类型

File: service/rpc/builder.go [hunk 2]
```diff
@@ -88,6 +89,9 @@ func (b *builder) SetModifier(m service.DefinitionModifier) {
 }
 
 func genRPCPath(path, version, action string) string {
+	if version == "" && action == "" {
```
Comment (xpofei) [inline service/rpc/builder.go:92]: 这个判断为空返回 path 是为了 swagger 好看吗

File: utils/api/definitions.go (hunk 1 above)
Comment (iawia002) [reply utils/api/definitions.go:100]: 没注意，看见 lint 报错直接改了

File: service/rpc/builder.go (hunk 2 above)
Comment (iawia002) [reply service/rpc/builder.go:92]: 对，对于内部的一些直接用 path 形式的插件来说，log 也好看点

```
INFO  1123-11:04:57.679+08 builder.go:184 | Path: /healthz, Consumes: [*/*], Produces: [*/*]
INFO  1123-11:04:57.679+08 builder.go:184 | Path: /metrics, Consumes: [*/*], Produces: [*/*]
```

**Pull request merged**